### DISCUSSION TITLE
Atlantis template, font broken

### DISCUSSION THREAD

**Discussion by popposoft** [primefaces/community#1632] [PrimeNG Templates] [answered]
Hello

I don't remember having this problem before, but I noticed you have this issue also in the preview theme published on https://atlantis.primeng.org/

All themes refer to "Inter var" font instead of the "Poppins" included and served by the application version 17.0.0

Additionaly, if possible, is there any way to build all included themes at once with one sass command?

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Thanks for reporting the issue, we'll check it and fix it before the upcoming template update in v18.
